cpp/src/strings/json/json_path.cu

```diff
@@ -171,7 +171,7 @@ class parser {
   char const* pos;
 
  private:
-  CUDA_HOST_DEVICE_CALLABLE bool is_whitespace(char c) { return c <= ' '; }
+  CUDA_HOST_DEVICE_CALLABLE bool is_whitespace(char c) { return c == ' '; }
```

Comment:
This change is incorrect. 
Whitespace is a superset of space which the `<=` captures. It includes `\n` ,`\r`, `\t` as well as other control characters.
You change means it will only check for space which I'm fairly sure is not intended. If that is desired, you should change the name to `is_space()`.

Reply:
Got it !